### ISSUE TITLE
UI-XAML files used outdated code for icons

### ISSUE THREAD

**Issue by Jay-o-Way** [microsoft/microsoft-ui-xaml#7350]
There are files that still use outdated codes for icons.
![image](https://user-images.githubusercontent.com/65828559/111816539-3f889e00-88dd-11eb-85f6-e9b0f0f65256.png)
https://docs.microsoft.com/windows/apps/design/style/segoe-fluent-icons-font

For example - but not limited to:
[microsoft-ui-xaml](https://github.com/microsoft/microsoft-ui-xaml/tree/6684208ed977a1490bce9a6d0a8722e21d6d77ec)/[dev](https://github.com/microsoft/microsoft-ui-xaml/tree/6684208ed977a1490bce9a6d0a8722e21d6d77ec/dev)/[NavigationView](https://github.com/microsoft/microsoft-ui-xaml/tree/6684208ed977a1490bce9a6d0a8722e21d6d77ec/dev/NavigationView)/NavigationView_rs1_themeresources.xaml

```xaml
397    <Setter Property="Content" Value="&#xE11A;"/>
464    Glyph="&#xE10C;"
532    Glyph="&#xE10C;"
```
Please check these files and update according to your own recommendation

**Comment by krschau** [comment]
I see rs1 in the name, and other files with older OS references in their names, which makes me suspect these files are used for those down-level OS versions where we _want_ the deprecated symbols.

Maybe @evelynwu-msft can confirm?

**Comment by Jay-o-Way** [comment]
I dunno about the significane of "rs1" or anything, but that message looks pretty clear to me

**Comment by Jay-o-Way** [comment]
@ranjeshj seeems to be on vacation, anybody else?

**Comment by Jay-o-Way** [comment]
ping @bpulliam

**Comment by pratikone** [comment]
I can look at it.

**Comment by Jay-o-Way** [comment]
@pratikone thanks, but don't bother, have a look at https://github.com/microsoft/microsoft-ui-xaml/issues/8375 and https://github.com/microsoft/microsoft-ui-xaml/pull/8379 :)

**Comment by pratikone** [comment]
Thanks for your contributions.

**Comment by Jay-o-Way** [comment]
Superseded by #8375 and (should be) fixed with #8379

**Comment by pratikone** [comment]
yes, thanks for mentioning it
